Library/EnvironmentVariables.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+public enum VariableName: String {
+  case koalaTracking = "KOALA_TRACKING"
+}
+
+public struct EnvironmentVariables {
+  private let processInfo: ProcessInfoType
+
+  public init(processInfo: ProcessInfoType = ProcessInfo.processInfo) {
+    self.processInfo = processInfo
+  }
+}
+
+extension EnvironmentVariables {
+  public var kaoalaTrackingIsOn: Bool {
```

Comment:
Another option for naming these could be `isKoalaTrackingEnabled`, `isNetworkLoggingEnabled`, etc.

Reply:
I like `isKoalaTrackingEnabled` 👍